concepts/admin.py

```diff
@@ -8,16 +11,46 @@ def resolve(modeladmin, request, queryset):
         authorities.resolve(type(obj), obj)
 resolve.verbose_name = 'resolve selected concepts'
 
+def merge_concepts(modeladmin, request, queryset):
+
+    opts = modeladmin.model._meta
+    app_label = opts.app_label
+    RESOLVED_STATUS = 'Resolved'
```

Comment:
Don't hard-code this; the status options are defined on the model itself. E.g. you can use ```Concept.RESOLVED```